[2023-01-03T20:07:30.896+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_pipeline.dashboard manual__2023-01-03T19:46:51.588373+00:00 [queued]>
[2023-01-03T20:07:30.910+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_pipeline.dashboard manual__2023-01-03T19:46:51.588373+00:00 [queued]>
[2023-01-03T20:07:30.911+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-03T20:07:30.912+0000] {taskinstance.py:1284} INFO - Starting attempt 4 of 4
[2023-01-03T20:07:30.912+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-03T20:07:30.932+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): dashboard> on 2023-01-03 19:46:51.588373+00:00
[2023-01-03T20:07:30.937+0000] {standard_task_runner.py:55} INFO - Started process 7161 to run task
[2023-01-03T20:07:30.943+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'accidents_pipeline', 'dashboard', 'manual__2023-01-03T19:46:51.588373+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/Airflow.py', '--cfg-path', '/tmp/tmpu085ounu']
[2023-01-03T20:07:30.945+0000] {standard_task_runner.py:83} INFO - Job 83: Subtask dashboard
[2023-01-03T20:07:31.026+0000] {task_command.py:389} INFO - Running <TaskInstance: accidents_pipeline.dashboard manual__2023-01-03T19:46:51.588373+00:00 [running]> on host 321b86f85270
[2023-01-03T20:07:31.132+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=accidents_pipeline
AIRFLOW_CTX_TASK_ID=dashboard
AIRFLOW_CTX_EXECUTION_DATE=2023-01-03T19:46:51.588373+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-03T19:46:51.588373+00:00
[2023-01-03T20:07:34.866+0000] {logging_mixin.py:137} INFO - Dash is running on http://0.0.0.0:8050/
[2023-01-03T20:07:34.866+0000] {dash.py:1968} INFO - Dash is running on http://0.0.0.0:8050/

[2023-01-03T20:07:34.876+0000] {logging_mixin.py:137} INFO -  * Serving Flask app '***'
[2023-01-03T20:07:34.877+0000] {logging_mixin.py:137} INFO -  * Debug mode: off
[2023-01-03T20:07:34.878+0000] {logging_mixin.py:137} WARNING - Address already in use
[2023-01-03T20:07:34.878+0000] {logging_mixin.py:137} WARNING - Port 8050 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2023-01-03T20:07:34.930+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-01-03T20:07:34.959+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
